src/main/kotlin/com/sourcegraph/cody/context/ui/ContextToolbarButton.kt

```diff
@@ -11,8 +11,8 @@ open class ContextToolbarButton(
     private val buttonAction: () -> Unit = {}
 ) : DumbAwareActionButton(name, icon) {
 
-  init {
```

Comment:
I started IU 2024.1 and got this:

```
2024-05-27 11:27:48,267 [   5427] SEVERE - #c.i.o.a.i.ActionUpdater - HelpButton#presentation@ToolbarDecorator (com.sourcegraph.cody.context.ui.HelpButton), text='Help'
com.intellij.diagnostic.PluginException: Cannot load class com.intellij.openapi.actionSystem.ActionUpdateThread (
  error: loader constraint violation: loader com.intellij.ide.plugins.cl.PluginClassLoader @7180c414 wants to load class com.intellij.openapi.actionSystem.ActionUpdateThread. A different class with the same name was previously loaded by com.intellij.util.lang.PathClassLoader @4563e9ab. (com.intellij.openapi.actionSystem.ActionUpdateThread is in unnamed module of loader com.intellij.util.lang.PathClassLoader @4563e9ab),
  classLoader=PluginClassLoader(plugin=PluginDescriptor(name=Cody: AI Coding Assistant with Autocomplete & Chat, id=com.sourcegraph.jetbrains, descriptorPath=plugin.xml, path=~\IdeaProjects\jetbrains\build\idea-sandbox\plugins\Sourcegraph, version=6.0-localbuild, package=null, isBundled=false), packagePrefix=null, state=active)
)
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClassInsideSelf(PluginClassLoader.kt:331)
	at com.intellij.ide.plugins.cl.PluginClassLoader.tryLoadingClass(PluginClassLoader.kt:178)
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.kt:151)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
	at java.base/java.lang.Class.getDeclaredMethods0(Native Method)
	at java.base/java.lang.Class.privateGetDeclaredMethods(Class.java:3402)
	at java.base/java.lang.Class.getMethodsRecursive(Class.java:3543)
	at java.base/java.lang.Class.getMethodsRecursive(Class.java:3557)
	at java.base/java.lang.Class.getMethod0(Class.java:3529)
	at java.base/java.lang.Class.getMethod(Class.java:2225)
	at com.intellij.util.ReflectionUtil.getMethodDeclaringClass(ReflectionUtil.java:203)
	at com.intellij.openapi.actionSystem.ActionClassMetaData.getFlags(ActionClassMetaData.java:55)
	at com.intellij.openapi.actionSystem.ActionClassMetaData.isDefaultUpdate(ActionClassMetaData.java:27)
	at com.intellij.openapi.actionSystem.impl.ActionUpdaterKt.isDefaultImplementationRecursively(ActionUpdater.kt:796)
	at com.intellij.openapi.actionSystem.impl.ActionUpdaterKt.access$isDefaultImplementationRecursively(ActionUpdater.kt:1)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater$updateAction$success$1$1.invokeSuspend(ActionUpdater.kt:560)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater$updateAction$success$1$1.invoke(ActionUpdater.kt)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater$updateAction$success$1$1.invoke(ActionUpdater.kt)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater$updateAction$$inlined$retryOnAwaitSharedData$1.invokeSuspend(ActionUpdater.kt:862)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater$updateAction$$inlined$retryOnAwaitSharedData$1.invoke(ActionUpdater.kt)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater$updateAction$$inlined$retryOnAwaitSharedData$1.invoke(ActionUpdater.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:78)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:167)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.updateAction(ActionUpdater.kt:888)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.expandGroupChild(ActionUpdater.kt:412)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater.access$expandGroupChild(ActionUpdater.kt:80)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater$doExpandActionGroup$2$expandResult$1$1$1.invokeSuspend(ActionUpdater.kt:339)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(LimitedDispatcher.kt:115)
	at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:103)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:584)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:793)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:697)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:684)
Caused by: java.lang.LinkageError: loader constraint violation: loader com.intellij.ide.plugins.cl.PluginClassLoader @7180c414 wants to load class com.intellij.openapi.actionSystem.ActionUpdateThread. A different class with the same name was previously loaded by com.intellij.util.lang.PathClassLoader @4563e9ab. (com.intellij.openapi.actionSystem.ActionUpdateThread is in unnamed module of loader com.intellij.util.lang.PathClassLoader @4563e9ab)
	at java.base/java.lang.ClassLoader.defineClass2(Native Method)
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1108)
	at com.intellij.util.lang.UrlClassLoader.consumeClassData(UrlClassLoader.java:291)
	at com.intellij.util.lang.ZipResourceFile.findClass(ZipResourceFile.java:116)
	at com.intellij.util.lang.JarLoader.findClass(JarLoader.java:58)
	at com.intellij.util.lang.ClassPath.findClassInLoader(ClassPath.java:240)
	at com.intellij.util.lang.ClassPath.findClass(ClassPath.java:190)
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClassInsideSelf(PluginClassLoader.kt:326)
	... 35 more
```

Reply:
Note I needed to change the code in order to get it compiling & running with 2024.1 and IU (in a dev run). Maybe if the plugin is built and installed on that spec, and run in a regular way there is no problem.

Reply:
false alarm - sorry. I changed gradle.properties to switch the version and the type. As I mentioned - built and installed works correctly 🚀 lgtm